src/native/eventpipe/ep.c

```diff
@@ -455,25 +450,25 @@ is_session_id_in_collection (EventPipeSessionID session_id)
 	return false;
 }
 
+static bool check_options_valid (const EventPipeSessionOptions *options)
+{
+	ep_return_false_if_nok (options->format < EP_SERIALIZATION_FORMAT_COUNT);
```

Comment:
nit: maybe use the same kind of error checking instead of mixing ep_return_false_if_nok and regular if (...) return false checks?

Reply:
Would you prefer explicit `if (...) return false` or writing a new macro like `ep_return_false_if_ok`? The expressions that use `if` currently are hard to read when converted to `false_if_nok`

Reply:
I would prefer easy read, so explicit if in this case.